N-Queens Problem/N-Queens.java

```diff
@@ -0,0 +1,56 @@
+import java.util.Scanner;
+public class NQueens 
```

Comment:
Rather than a directory for itself, you can add it to the Leetcode directory since this is a leetcode problem iirc.

Reply:
Okay, N-Queens #101 problem has been added in Leetcode folder.
**commit 0f424794a40554561f8b6c81d957f9c1ae65a94d**
@Pradyuman7